lxd/storage/drivers/driver_ceph_volumes.go

```diff
@@ -986,7 +986,7 @@ func (d *ceph) SetVolumeQuota(vol Volume, size string, allowUnsafeResize bool, o
 
 // GetVolumeDiskPath returns the location of a root disk block device.
 func (d *ceph) GetVolumeDiskPath(vol Volume) (string, error) {
-	if vol.IsVMBlock() || (vol.volType == VolumeTypeCustom && vol.contentType == ContentTypeBlock) {
+	if vol.volType == VolumeTypeCustom && vol.contentType == ContentTypeBlock {
```

Comment:
This whole commit `lxd/storage/drivers/ceph: Do not mount block volumes` looks problematic to me. 
How will LXD manage the contents of the block volume (thinking create, copy, backup etc) if it doesn't activate the RBD block device?

Have you run this PR through:

* https://github.com/lxc/lxc-ci/blob/master/bin/test-lxd-storage-disks-vm
* https://github.com/lxc/lxc-ci/blob/master/bin/test-lxd-storage-vm
* https://github.com/lxc/lxc-ci/blob/master/bin/test-lxd-storage-volumes-vm

Reply:
Chatting with @stgraber we think we need to provide the ability for the storage subsystem to differentiate between needing to mount both the instance config and rootfs volumes and just needing to mount the instance config volume.

Suggest adding a `(b *lxdBackend) MountInstanceConfig(inst instance.Instance, op *operations.Operation) error` function (and associated unmount function) , which for VMs will just pass the filesystem volume to the storage driver to mount. Then this can be used from the QEMU instance driver when starting the instance, rather than `MountInstance()`.